--- a/src/sdl2/video.c
+++ b/src/sdl2/video.c
@@ -67,7 +67,8 @@
 #if defined(__SWITCH__) || defined(__vita__) || defined(__PSP__) || defined(__PSL1GHT__) || defined(__WIIU__)
                                SDL_WINDOW_FULLSCREEN);
 #else
-                               0);
+                               SDL_WINDOW_FULLSCREEN_DESKTOP);
+//                               0);
 #endif
 
     if (gWindow == NULL)
@@ -97,8 +98,11 @@
         exit(1);
     }
 
-    Uint32 format = SDL_PIXELFORMAT_RGB24;
+//    Uint32 format = SDL_PIXELFORMAT_RGB24;
+    Uint32 format = SDL_PIXELFORMAT_BGR24;
 
+//    Uint32 format = SDL_PIXELFORMAT_RGB24;
+
     // HACK: this is needed for my crappy SDL2 implementation (https://github.com/sergiou87/SDL2/commit/962e4e565562c2cd70b877f3d697ad2084d9405b)
     // but this should be fixed on SDL's side (or pspgl's), I think.
     //
@@ -118,7 +122,7 @@
         exit(1);
     }
 
-    gTextureSurface = SDL_CreateRGBSurfaceWithFormat(0, kScreenWidth, kScreenHeight, 8, SDL_PIXELFORMAT_RGB24);
+    gTextureSurface = SDL_CreateRGBSurfaceWithFormat(0, kScreenWidth, kScreenHeight, 8, SDL_PIXELFORMAT_BGR24);
 
     if (gTextureSurface == NULL)
     {
